lib/src/main/java/edu/utexas/tacc/tapis/files/lib/clients/GlobusDataClient.java

```diff
@@ -476,10 +474,8 @@ else if (globusFilesList.size() != 1)
     fileInfo.setLastModified(convertLastModified(globusFileInfo.getLastModified()));
     fileInfo.setNativePermissions(globusFileInfo.getPermissions());
     if (globusFileInfo.getSize() != null) fileInfo.setSize(globusFileInfo.getSize());
-    // Set the type
-    // NOTE: currently can use the type from Globus because they match our defined types of
```

Comment:
I think this comment meant that what we get back from the globus proxy client for type might potentially be different from what Files normally uses. Maybe that is OK.